install_config/topics/manually_configuring_for_gce.adoc

```diff
@@ -0,0 +1,126 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='gce-configuring-masters-manual_{context}']
+= Option 2: Manually configuring {product-title} for GCE
+
+== Manually configuring master hosts for GCE
+
+Perform the following on all master hosts.
+
+.Procedure
+
+. Edit the master configuration file at `/etc/origin/master/master-config.yaml`
+ by default on all masters and update the contents of the `apiServerArguments`
+ and `controllerArguments` sections:
++
+[source,yaml]
+----
+kubernetesMasterConfig:
+  ...
+  apiServerArguments:
+    cloud-provider:
+      - "gce"
+    cloud-config:
+      - "/etc/origin/cloudprovider/gce.conf"*
+  controllerArguments:
+    cloud-provider:
+      - "gce"
+    cloud-config:
+      - "/etc/origin/cloudprovider/gce.conf"*
+----
+
+. When you configure {product-title} for GCP using Ansible, the
```

Comment:
Why not just "Create a file named  *_ file _* that contains the following section:" ?